guides/common/modules/proc_configuring-repositories-proxy.adoc

```diff
@@ -5,9 +5,6 @@
 Use this procedure to enable the repositories that are required to install {ProductName}.
 Select the option that corresponds with operating system and version you want to install on:
```

Comment:
Perhaps we could remove this line as well, when we support only a single version?

Reply:
Addressed :+1: